packages/lib-user/src/components/shared/GroupContainer/components/DeactivatedGroup.js

```diff
@@ -0,0 +1,57 @@
+import { Loader, SpacedText } from '@zooniverse/react-components'
+import { Box } from 'grommet'
+import { SubtractCircle } from 'grommet-icons'
+import { bool, func, string } from 'prop-types'
+
+import { HeaderButton } from '@components/shared'
+
+const DEFAULT_HANDLER = () => true
+
+function DeactivatedGroup({
+  deleteMembership = DEFAULT_HANDLER,
+  deleteMembershipLoading = false,
+  membershipId
+}) {
+  async function handleGroupMembershipLeave ({
+    membershipId
+  }) {
+    await deleteMembership({ membershipId }, {
+      revalidate: true
+    })
+  
+    window.location.href = '/'
+  }
+
+  return (
+    <Box
+      align='center'
+      direction='column'
+      gap='medium'
+    >
+      <SpacedText uppercase={false}>
+        This is a deactivated group.
+        Please leave the group or contact the group administrator if you believe this is an error.
```

Comment:
I'm not sure about this message, I think it's ok, but let me know what you think.